nat-lab/tests/uniffi/libtelio_remote.py

```diff
@@ -5,12 +5,17 @@
 import Pyro5.server  # type: ignore
 import sys
 import telio_bindings as libtelio  # type: ignore # pylint: disable=import-error
+from serialization import (  # type: ignore # pylint: disable=import-error
+    init_serialization,
+)
 from threading import Lock
 from typing import List
 
 REMOTE_LOG = "remote.log"
 TCLI_LOG = "tcli.log"
 
+init_serialization(libtelio)
```

Comment:
Could you add a small comment here and in `libtelio_proxy` on what this function is doing. A small 1-liner so that anyone going working on the code can understand as just by reading it doesn't feel exactly clear what this would do.